src/ray/stats/metric_defs.h

```diff
@@ -113,20 +113,22 @@ static Histogram ProcessStartupTimeMs("process_startup_time_ms",
                                       "Time to start up a worker process.", "ms",
                                       {1, 10, 100, 1000, 10000});
 
-static Gauge AvgNumScheduledTasks(
-    "avg_num_scheduled_tasks",
-    "Number of tasks that are queued on this node per second. It doesn't guarantee "
-    "that the task will be executed in this node. If the task is executed, it is "
-    "recorded as avg_num_executed_tasks. If the task is not executed and needs to be "
-    "scheduled in other nodes, it will be recorded as avg_num_spilled_back_tasks",
-    "tasks");
-
-static Gauge AvgNumExecutedTasks("avg_num_executed_tasks",
-                                 "Number of executed tasks on this node per second.",
-                                 "tasks");
-
-static Gauge AvgNumSpilledBackTasks("avg_num_spilled_back_tasks",
-                                    "Number of spilled back tasks per second.", "tasks");
+static Sum NumWorkersStarted(
+    "internal_num_processes_started",
+    "The total number of worker processes the worker pool has created.", "processes");
+
+static Sum NumReceivedTasks(
+    "internal_num_received_tasks",
+    "The cumulative number of lease requeusts that this raylet has received.", "tasks");
+
+static Sum NumDispatchedTasks(
+    "internal_num_dispatched_tasks",
+    "The cumulative number of lease requeusts that this raylet has granted.", "tasks");
+
+static Sum NumSpilledTasks("internal_num_spilled_tasks",
+                           "The cumulative number of lease requeusts that this raylet "
+                           "has spilled to other raylets.",
+                           "tasks");
```

Comment:
Are these all represented sensibly once in Prometheus?

Reply:
They do appear to be... (I've verified the prometheus end point looks right).